### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by bernd-wechner** [fmoralesc/pastie#49]
Sure looks that way. Last minor touch 7 years ago, last significant one 11 years ago. No Readme. 18 open issues.

A python solution looks nice to me, gave it a quick whir but already lost on `import gtk` and googling solutions leaves me moving on alas. A shame. None of the forks are any more up to date or well documented either.

**Comment by fmoralesc** [comment]
Indeed. I have moved myself on to some of the current alternatives (gpaste and more recently, gnome-shell extensions). The mechanism that these things works on is very simple, and other than updating the Gtk calls (the clipboard API has remained pretty much stable across Gtk versions), there is not much to do to update, but there has not been much incentive to do so.
